### PR TITLE
Add actions to mass edit Administation ACL permissions

### PR DESCRIPTION
**Pull request by R0MANDEV**
#### Type Of Change 
- [x] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Select multiple entities and do a bulk edit

### CODE REVIEW

File: web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/RevokeAll.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    }
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+    setError(false);
+  };
+
+  const admId =
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/RevokeAll.tsx:55]: same here

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

File: web/portal/brand/src/entities/AdministratorRelPublicEntity/Action/RevokeAll.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        setOpen(false);
+
+        if (resp !== undefined) {
+          setTimeout(() => {
+            window.location.reload();
+          });
+        }
+      })
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      .catch((error: Record<string, any>) => {
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/AdministratorRelPublicEntity/Action/RevokeAll.tsx:88]:
```suggestion
      .catch((error: ApiError) => {
```
Defined in ivoz-ui/services/api/ApiClient.ts

File: web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/GrantAll.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        setOpen(false);
+
+        if (resp !== undefined) {
+          setTimeout(() => {
+            window.location.reload();
+          });
+        }
+      })
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      .catch((error: Record<string, any>) => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/GrantAll.tsx:88]:
```suggestion
      .catch((error: ApiError) => {
```

File: web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/GrantAll.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+      values: payload,
+      contentType: 'application/json',
+      silenceErrors: true,
+    })
+      .then(() => {
+        setOpen(false);
+
+        if (resp !== undefined) {
+          setTimeout(() => {
+            window.location.reload();
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/GrantAll.tsx:83]: use `store.list.reload()`

File: web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/GrantReadOnly.tsx [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+      values: payload,
+      contentType: 'application/json',
+      silenceErrors: true,
+    })
+      .then(() => {
+        setOpen(false);
+
+        if (resp !== undefined) {
+          setTimeout(() => {
+            window.location.reload();
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/GrantReadOnly.tsx:83]: use `store.list.reload()`

File: web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/GrantReadOnly.tsx [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+          });
+        }
+      })
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      .catch((error: Record<string, any>) => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/GrantReadOnly.tsx:88]:
```suggestion
      .catch((error: ApiError) => {
```

File: web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/RevokeAll.tsx [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+      values: payload,
+      contentType: 'application/json',
+      silenceErrors: true,
+    })
+      .then(() => {
+        setOpen(false);
+
+        if (resp !== undefined) {
+          setTimeout(() => {
+            window.location.reload();
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/RevokeAll.tsx:83]: same here

File: web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/RevokeAll.tsx [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+          });
+        }
+      })
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      .catch((error: Record<string, any>) => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/AdministratorRelPublicEntity/Action/RevokeAll.tsx:88]: same here

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

File: web/portal/brand/src/translations/es.json [hunk 9]
```diff
@@ -305,6 +308,8 @@
   "Global": "Global",
   "Go to Carriers": "Ir a Carriers",
   "Go to DDIs": "Ir a DDIs",
+  "Grant Read Only Access": "Acceso de solo lectura",
+  "Grant Write Access": "Acceso de escriptura",
```
Comment (Kaian) [inline web/portal/brand/src/translations/es.json:312]: _Acceso de escritura_

File: web/portal/platform/src/translations/ca.json [hunk 10]
```diff
@@ -1,5 +1,6 @@
 {
   "'https://' URLs valid only (without trailing '/')": "Només s'accepten adreces 'https://' (sense '/' al final)",
+  "Accept": "AcceptarWhite",
```
Comment (Kaian) [inline web/portal/platform/src/translations/ca.json:3]: AcceptarWhite?

File: web/portal/platform/src/translations/es.json [hunk 11]
```diff
@@ -114,6 +118,8 @@
   "Global Special Number_one": "Número especial Global",
   "Global Special Number_other": "Números especiales Globales",
   "Go to brands": "Ir a marcas",
+  "Grant Read Only Access": "Acceso de solo lectura",
+  "Grant Write Access": "Acceso de escriptura",
```
Comment (Kaian) [inline web/portal/platform/src/translations/es.json:122]: escritura

**Review by Kaian — CHANGES_REQUESTED** [review]

File: web/portal/platform/src/translations/ca.json (hunk 10 above)
Comment (Kaian) [reply web/portal/platform/src/translations/ca.json]: You have included this platform portal change in a commit tagged as portal/brand

File: web/portal/platform/src/translations/es.json (hunk 11 above)
Comment (Kaian) [reply web/portal/platform/src/translations/es.json]: You have included this platform portal change in a commit tagged as portal/brand

**Comment by Kaian** [comment]
Code is not properly split in commits.

portal/brand commit contains changes that belongs to portal/platform.

**Review by Kaian — APPROVED** [review]

**Pull request merged**